### ISSUE TITLE
NTR: EBV acute infection test positive (Lab: serology & immunological tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8714]
New term request EBV acute infection test positive (elthat_100424164919): Positive monospot test; heterophile-agglutinating antibodies; Epstein–Barr virus PCR or IgM antibodies  UMLS: 5203865

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000923
- label: Positive bloodstream EBV monospot test
- parents: [Positive bloodstream pathogen-specific antibody positivity (HP:0430051)](https://hpo.jax.org/app/browse/term/HP:0430051)
- pmid: PMID:30969561
- def: The monospot test is a latex agglutination test which utilizes equine erythrocytes as the primary substrate and tests for specific heterophile antibodies produced by the human immune system in response to EBV infection. When these specific antibodies are present in the patient's blood specimen, exposure to equine erythrocytes will lead to clumping of the sample, thus signaling a positive agglutination reaction.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8714
- synonyms:
